blackbox/docs/sql/reference/create_ingest_rule.txt

```diff
@@ -52,16 +52,21 @@ Notes
     The default user for the ``INSERT`` operations in the target
     ``table_ident`` is the superuser ``crate``.
 
-If no ingestion plugin is registered, the ingestion rules will not take any
-effect.
+Ingestion rules can be created and dropped, even if no ingestion plugin is
+available. In such case the ingestion rules will not take any effect, until
+the plugin is registered.
 
-There is no default ingestion rule, all incoming messages into any ingestion
-endpoint are discarded by default.
+There is no default ingestion rule, all incoming streams into any ingestion
+endpoint are discarded by default. This is a contract every plugin should
+follow, but it's not under CrateDB's control.
```

Comment:
I'd say: "... all incoming streams into any ingestion endpoint should be discarded by default by any plugin implementation."